doc/hexyl.1.md

```diff
@@ -0,0 +1,145 @@
+% HEXYL(1) hexyl 0.9.0 | General Commands Manual
+%
+% 2022-03-23
+
+<!--
+This man page is written in Pandoc's Markdown.
+See: https://pandoc.org/MANUAL.html#pandocs-markdown
+-->
+
+# NAME
+
+hexyl - a command-line hex viewer
+
+# SYNOPSIS
+
+**hexyl** [_OPTIONS_] [_FILE_]
+
+# DESCRIPTION
+
+**hexyl** is a simple hex viewer for the terminal.
+It uses a colored output to distinguish different categories of bytes (NULL
+bytes, printable ASCII characters, ASCII whitespace characters, other ASCII
+characters and non-ASCII).
+
+# POSITIONAL ARGUMENTS
+
+_FILE_
+:   The file to display.
+    If no _FILE_ argument is given, read from STDIN.
+
+# OPTIONS
+
+**-n**, **\--length** _N_
+:   Only read _N_ bytes from the input.
+    The _N_ argument can also include a unit with a decimal prefix (kB, MB, ..)
+    or binary prefix (kiB, MiB, ..), or can be specified using a hex number.
+
+    Examples:
+
+    :   
+
+        Read the first 64 bytes:
+        :   $ **hexyl \--length=64**
+
+        Read the first 4 kibibytes:
+        :   $ **hexyl \--length=4KiB**
+
+        Read the first 255 bytes (specified using a hex number):
+        :   $ **hexyl \--length=0xff**
+
+**-c**, **\--bytes** _N_
+:   An alias for **-n**/**\--length**.
+
+**-l** _N_
+:   Yet another alias for **-n**/**\--length**.
+
+**-s**, **\--skip** _N_
+:   Skip the first _N_ bytes of the input.
+    The _N_ argument can also include a unit (see **\--length** for details).
+    A negative value is valid and will seek from the end of the file.
+
+**\--block-size** _SIZE_
+:   Sets the size of the block unit to _SIZE_ (default is 512).
+
+    Examples:
+
+    :   
+
+        Sets the block size to 1024 bytes:
+        :   $ **hexyl \--block-size=1024 \--length=5block**
+
+        Sets the block size to 4 kilobytes:
+        :   $ **hexyl \--block-size=4kB \--length=20blocks**
```

Comment:
Sorry. One more think. block**s** doesn't work, unfortunately. We should either make that work (I'd think it would be relatively easy) or change the man page here once more.